File: class/java/home/ex_home/session_one/demo29.java
import java.util.Scanner;

public class demo29 {
    public static void main(String[] agrs){
        Scanner scanner = new Scanner(System.in);

        System.out.print("Input Day: ");
        int n = scanner.nextInt();

        int year = n/365;
        int week = (n-(year*365))/7;
        int day = n-(year * 365) - (week * 7);

        System.out.println("Year: "+year+"\nWeek: "+week+"\nDay: "+day);
    }
}

File: python/ex_basic/Session_one/demo29.py
print("Day Number Input: ", end=" ")
day_number = int(input())
years = day_number // 365
weeks = (day_number-(years*365))//7
days = day_number-((years*365)+(weeks*7))
print("Year = {}\nWeek = {}\nDay = {}".format(years, weeks, days))
